examples/TEM_data/tests/test_composition.py

```diff
@@ -0,0 +1,9 @@
+import dlite
+
+from paths import indir
+
+
+compfile = indir / "composition.csv"
+
+comp = dlite.Instance.from_location("composition", compfile)
+print(comp)
```

Comment:
printing in a test does not really test anything. Better to have some asserts. 

Also, I really think it is better to have properly separated tests in functions, but you know that...

Reply:
Added assert statements.

I agree that test functions have their use cases. But in simple tests like this, I find simple test scripts highly preferable since they make it easy to debug and investigate the local/global state by running it with `ipython -i`. They are also easy to understand with no hidden magic that pytest test functions tends to have.

Unless the user carefully document their pytest test functions, it is impossible to know whether an argument is assigned by some defaults within the pytest library or by a conftest.py function somewhere in your test directory. This makes it really difficult to understand what is in the local scope in the pytest test functions. You can not even investigate the local scope by put in print statements in the pytest functions, since pytest strips them away.

Reply:
I don't agree. You need to define the arguments in the function, and it is clear from what you import wether it is from a local conftest.py or not. 

Also, it s no problem to use print statements in pytest. Just run pytest with -rP.

Reply:
Thanks for the tip about the -rP option. To my knowledge, the assignment of test function arguments (called test fixtures by pytest) is done magically by pytest and is not related to imports.